web/web/src/app/metalakes/metalake/rightContent/RightContent.js

```diff
@@ -25,23 +25,38 @@ import { Box, Button, IconButton } from '@mui/material'
 import Icon from '@/components/Icon'
 import MetalakePath from './MetalakePath'
 import CreateCatalogDialog from './CreateCatalogDialog'
+import CreateSchemaDialog from './CreateSchemaDialog'
 import TabsContent from './tabsContent/TabsContent'
 import { useSearchParams } from 'next/navigation'
+import { useAppSelector } from '@/lib/hooks/useStore'
 
 const RightContent = () => {
   const [open, setOpen] = useState(false)
+  const [openSchema, setOpenSchema] = useState(false)
   const searchParams = useSearchParams()
   const [isShowBtn, setBtnVisiable] = useState(true)
+  const [isShowSchemaBtn, setSchemaBtnVisiable] = useState(false)
```

Comment:
I think it should be "Visible", right?

Reply:
It's visible when view the schema page, it's unvisible when view other pages. 
The judgment condition is implemented in lines 54 to 57.

Reply:
I mean "visiable" is a typo we need to fix.